app/assets/stylesheets/elements/_codemirror.scss

```
@@ -113,6 +113,11 @@
   .cm-searchMatch {
     background: lighten($bg-dark, 10%);
     box-shadow: 0 0 0 1px $primary;
+	
+    &-selected {
+      background: darken($bg-dark, 10%);
+      box-shadow: 0 0 0 1px $shadow;
+    }
   }
```

While classes like this work, they make it hard to search for. With BEM it's easier since you know the block level, but codemirror does not use bem.

```suggestion
  .cm-searchMatch {
    background: lighten($bg-dark, 10%);
    box-shadow: 0 0 0 1px $primary;
  }
	
  .cm-searchMatchselected {
    background: darken($bg-dark, 10%);
    box-shadow: 0 0 0 1px $shadow;
  }
```